packages/gasket-plugin-elastic-apm/README.md

```diff
@@ -4,13 +4,13 @@ Adds Elastic APM instrumentation to your application
 
 ## Installation
 
-#### New apps
+### New apps
```

Comment:
Leave these example sections at `####` please to keep consistent. Using level 4 heading keeps example sections from crowding the docsify sidebar menu on [gasket.dev](https://gasket.dev/#/packages/gasket-plugin-elastic-apm/README).

```suggestion
#### New apps
```